experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt

```diff
@@ -58,7 +58,7 @@ fun ApplicationScope.ImageViewerDesktop() {
         title = "Image Viewer",
         state = WindowState(
             position = WindowPosition.Aligned(Alignment.Center),
-            size = getPreferredWindowSize(800, 1000)
+            size = getPreferredWindowSize(720, 857)
```

Comment:
Why this size?

Reply:
Height is equivalent to iOS, I tried to find a width that visually seems harmonic on desktop.